### PR TITLE
feat: handle double tap for mobile version

### PR DESCRIPTION
**Pull request by KirillBobkov**

### CODE REVIEW

File: src/chart/inputlisteners/canvas-input-listener.component.ts [hunk 1]
```diff
@@ -232,7 +232,7 @@ export class CanvasInputListenerComponent extends ChartBaseElement {
 				let timeoutId: number | null = null;
 				return () => {
 					if (timeoutId) {
-						this.dbClickSubject.next(this.currentPoint);
```
Comment (DeltaZN) [inline src/chart/inputlisteners/canvas-input-listener.component.ts:235]: It's intersting that Anastasia tried to subscribe to `dbClickSubject` before and it didn't work

**Review by disyakidneyshot — COMMENTED** [review]
@KirillBobkov are you sure that these changes make sense? It looks like it's just another subject with no difference from dbClickSubject.

**Comment by KirillBobkov** [comment]
> 
Now there is a new method `observeDbTouch`, so it can be useful for external developers.
I've checked functionality in Chrome Devtools (Mobile version), and it works as expected.
Any ideas how to check in on IOS directly?

**Comment by disyakidneyshot** [comment]
@KirillBobkov But what's the difference between `observeDbClick`? Mobile users can successfully use `observeDbClick` for this, like it was. Or you mean just convenience in naming? If so I guess `observeDbTap` is better the `touch`

**Comment by KirillBobkov** [comment]
Anastasia asked about  `touch `naming, but if `tap `is more preferable, let's do `tap`
image

**Review by DeltaZN — APPROVED** [review]

**Review by disyakidneyshot — APPROVED** [review]

**Pull request merged**